examples/with-jest-jsdom/tests/setup.ts

```diff
@@ -1,9 +1,18 @@
 import '@testing-library/jest-dom/jest-globals';
 
 import { beforeAll, afterEach, afterAll } from '@jest/globals';
+import { httpInterceptor } from 'zimic/interceptor/http';
 
 import githubInterceptor from './interceptors/github';
 
+httpInterceptor.default.onUnhandledRequest(async (request, context) => {
+  const url = new URL(request.url);
+
+  if (url.hostname !== '127.0.0.1') {
+    await context.log();
+  }
+});
```

Comment:
The main purpose of #406 is to properly ignore unhandled requests to localhost because the test environment makes such requests and the logs could be polluted. In browser examples, we don't have this problem.

Therefore, could you remove this call from browser examples? They are `with-jest-jsdom`, `with-vitest-browser`, and `with-vitest-jsdom`.

```suggestion
```

Sorry if the issue wasn't completely clear!